frontend/src/views/DeclaredElementPage/index.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <div>
+    <DsfrNotice title="En construction" desc="Des nouvelles fonctionnalités arrivent bientôt !" />
+    <DsfrNotice
+      title="NOUVEL INGRÉDIENT"
+      desc="Ingrédient non intégré dans la base de donnée et en attente de validation. "
+    />
+    <div class="fr-container">
+      <DsfrBreadcrumb class="mb-8" :links="breadcrumbLinks" />
+      <div v-if="element" class="grid md:grid-cols-2 gap-4">
+        <div class="bg-grey-975 p-4 mb-8">
+          <p class="text-blue-france-sun-113">
+            <v-icon :name="icon" />
+            {{ typeName }}
+          </p>
+          <div class="grid grid-cols-3">
+            <p :aria-hidden="true" class="fr-h2">{{ authorizationInfo.flag }}</p>
+            <p class="content-center col-span-2">{{ authorizationInfo.text }}</p>
+          </div>
+          <div v-for="(info, idx) in elementProfile" :key="idx" class="grid grid-cols-3">
```

Comment:
Je conseillerais de rendre un peu plus unique l'id de la boucle for. Dans la même page on risque d'avoir d'autres `el` générés à partir d'un `v-for` (par ex la liste d'élements de l'autocomplete, etc). Pour éviter des soucis de réactivité côté Vue c'est mieux si chaque boucle `v-for` génère des ids différents.
```suggestion
          <div v-for="(info, idx) in elementProfile" :key="`element-profile-${idx}`" class="grid grid-cols-3">
```